app/components/home-page/organization-dashboard/index.scss

```diff
@@ -0,0 +1,16 @@
+.organization-dashboard-root {
+  width: 100%;
+
+  .dashboard-main {
+    width: calc(100% - 250px);
+    margin-left: 250px;
+    padding: 0.5em;
+    box-sizing: border-box;
+
+    &::before {
```

Comment:
Why is this favored over `padding-top`?

Reply:
No particular reason earlier implementation had an element for spacing I just removed it and added a pseudo-element